Samples/macOS-SPM-CommandLine/Package.swift

```diff
@@ -12,7 +12,7 @@ let package = Package(
     targets: [
         .target(
             name: "macOS-SPM-CommandLine",
-            dependencies: ["Sentry"], 
+            dependencies: ["Sentry", .product(name: "SentrySwiftUI", package: "Sentry")],
```

Comment:
`h`: I think instead of this change you need to add `.package(name: "SentrySwiftUI", url: "https://github.com/getsentry/sentry-cocoa", .branch("main") )`
https://github.com/getsentry/sentry-cocoa/blob/19ab746a586e914c463dcbaa2f4b3ff21a79b4ce/Samples/macOS-SPM-CommandLine/Package.swift#L10

Only then CI validates against the current commit
https://github.com/getsentry/sentry-cocoa/blob/19ab746a586e914c463dcbaa2f4b3ff21a79b4ce/.github/workflows/build.yml#L123-L129

Reply:
>  I think instead of this change you need to add .package(name: "SentrySwiftUI", url: "https://github.com/getsentry/sentry-cocoa", .branch("main") ) below

Swift package does not allow two packages with the same url.

By expliciting using `.product(name: "SentrySwiftUI", package: "Sentry")` SPM uses the same files from the available Sentry package that will have its `.brach("main")` replaced.

Reply:
TIL, thanks for the explanation 🙏